openshift/template.yaml

```diff
@@ -151,4 +153,9 @@ parameters:
   displayName: Query administration region
   required: false
   name: QUERY_ADMINISTRATION_REGION
-  value: ""
\ No newline at end of file
+  value: ""
+- description: Data replication factor at database level. 
+  displayName: Database Replication Factor
+  required: true
```

Comment:
if default value is provided then `required` shall be removed.

Reply:
kept required to keep it consistent. Changed its value to `false.`